syft/pkg/cataloger/common/generic_cataloger_test.go

```diff
@@ -26,17 +26,16 @@ func parser(_ string, reader io.Reader) ([]*pkg.Package, []artifact.Relationship
 }
 
 func TestGenericCataloger(t *testing.T) {
-
 	globParsers := map[string]ParserFn{
-		"**/a-path.txt": parser,
+		"**/*.txt": parser, // this glob should capture all files, including the empty one
```

Comment:
Can you explain this change? I want to understand your thinking here

Reply:
Yes, this glob (`**/*.txt`) will find a file named `empty.txt`, that will turn into an invalid package, and relationship, later on dropped from the final list. The previous glob (`**/a-path.txt`) wouldn't find `empty.txt`.